specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          "type": "string",
+          "description": "The provisioning state of the application. For example, Running",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The name of the application. For example, ApplicationTest1"
+        },
+        "memberOf": {
+          "title": "memberOf",
+          "$ref": "#/definitions/CreateApplicationMemberOfProperties"
+        }
+      }
+    },
+    "Application": {
+      "description": "The application details.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource application."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "properties": {
+          "title": "properties",
+          "$ref": "#/definitions/ApplicationProperties"
+        }
+      }
+    },
+    "ApplicationProperties": {
+      "description": "Application creation request body parameters.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the application. For example, Running",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The name of the application. For example, ApplicationTest1"
+        },
+        "memberOf": {
+          "title": "memberOf",
+          "$ref": "#/definitions/CreateApplicationMemberOfProperties"
+        }
+      }
+    },
+    "CreateApplicationMemberOfProperties": {
+      "description": "The details of the parent management group or application.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID of the parent management group or applications.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000 or /providers/Microsoft.Management/applications/TestApplication"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "GroupIdParameter": {
```

Comment:
>GroupIdParameter [](http://example.com/codeflow?start=5&length=16)

Get this from ManagementGroupNameParameter definition in v5 common types. Also, IIUC the MG name is not really an ID, since it isn't unique. If that's the case, then "name" is a better moniker than "ID" for names and descriptions.